### DISCUSSION TITLE
viseron advancement

### DISCUSSION THREAD

**Discussion by spupuz** [roflcoopter/viseron#668] [General]
is the project stuck? i see no new update since months,

i like your project and i hope it will have frequemtly updates.

**Comment by roflcoopter** [comment]
It most definitely is not! 😊 I am working hard on the next version where the main feature is 24/7 recordings.
You can follow the progress here: #619, the latest commit was only 5 days ago.

It does take a lot of time to do stuff like this as a solo developer with a full time job and other obligations.

I could try to get better att making small releases tho, i often want to do a bit too much before releasing which is not always great for the users

**Comment by spupuz** [comment]
thanks for the fast response

**Comment by roflcoopter** [comment]
Actually i think i will release what is currently in `dev`, thanks for nudging me! 😁

**Comment by spupuz** [comment]
hello there hope you are fine any good news about release?

**Comment by roflcoopter** [comment]
Sorry for the delay, i actually became a father late september so its been hard to find enough time.
I am making slow but steady progress tho!

Here are some screenshots which shows the new events/timeline page
![Events](https://github.com/roflcoopter/viseron/assets/26493864/c0a48440-b3ed-45c7-8a69-77b76ca7a429)
![Timeline](https://github.com/roflcoopter/viseron/assets/26493864/09e29432-b1b3-49eb-87d5-292137fae90c)

The pink portion of the timeline shows active motion, and the images are of detected objects (dummy images atm tho)

Its not quite ready for release yet so bear with me on this one! It will be awesome once its ready :)

**Reply by spupuz** [reply]
Looks interesting, don't worry I'm father too, my little girl just turned 2 in October ;}

**Reply by roflcoopter** [reply]
Ah nice, then you know exactly how it is 😆
